oracle_server/oracle_server.go

```diff
@@ -472,7 +473,7 @@ func (os *OracleServer) buildRoundData(round uint64) (*types.RoundData, error) {
 	for _, s := range os.protocolSymbols {
 		p, err := os.aggregatePrice(s, int64(os.curSampleTS))
 		if err != nil {
-			os.logger.Error("aggregatePrice", "error", err.Error(), "symbol", s)
+			os.logger.Warn("aggregatePrice", "error", err.Error(), "symbol", s)
```

Comment:
> If this is a warning. Maybe we should not say error in the message itself. 'price aggregation failed due to "err.Error()"'. Also it should only be warning if there is something to fix/actionable item from end-user point of view. Else this can be a debug

Resolved in bd07764ac76b51a6cd8ae44a169d0b74eb116d63